Readme.md

```diff
@@ -15,33 +15,117 @@ I recommend you to use [Atom](https://atom.io/) as your text editor, it's free.
 To run Javascript outside of your browser, we need something called "Node". Node is a JavaScript runtime, we'll teach you more about it later.
 
 ##### 3. Git
-This whole project is managed with Git. Git is a version control system, it allows people to collaborate and it also allows to track who did what. Simple.
+This whole project is managed with [Git](https://git-scm.com/). Git is a version control system, by keeping track of historical points of the project, allow people to collaborate and it also allows to track who did what. Simple.
 
 I recommend you to use [GitHub Desktop](https://desktop.github.com/) for this, as it has a painless integration with GitHub.
 
 ### Cloning the repo
 
+#### ...with GitHub Desktop
+> TODO: @Gerd, add steps for GitHub Desktop
 
+> TODO: failed attempt of Jessica trying to explain how to do this on the command line... this is too difficlut to summarize as "simply do A, then execute B, you're done with C now..."
```

Comment:
also, good job there, feel free to continue

Reply:
*difficult*, typo there...